Source/WebCore/Modules/webcodecs/OpusEncoderConfig.h

```diff
@@ -54,7 +54,7 @@ struct OpusEncoderConfig {
     using BitstreamFormat = OpusBitstreamFormat;
     OpusBitstreamFormat format { OpusBitstreamFormat::Opus };
     uint64_t frameDuration { 20000 };
-    std::optional<size_t> complexity;
+    std::optional<size_t> complexity { 9 };
```

Comment:
Why optional then?

Reply:
Indeed it doesn't have to be if we set a default. In theory it should be 9 of desktop and 5 is mobile. I thought it was a fingerprint issue, so set it to 9 all the time